app/views/components/docs/action-link.yml

```diff
@@ -0,0 +1,36 @@
+name: Action link
+description: Use action links to help users get to the next stage of a journey quickly.
+shared_accessibility_criteria:
+  - link
+examples:
+  default:
+    data:
+      text: Go to NHS 111 online
+      href: '/my-test-page'
+  with_no_wrapping_text:
+    data:
+      text: "Coronavirus (COVID-19):"
+      nowrap_text: what you need to do 
+      href: '/my-test-page'
+  with_light_text:
+    data:
+      text: "Coronavirus (COVID-19):"
+      nowrap_text: what you need to do 
```

Comment:
Maybe remove the `nowrap_text` from this example, as it's not the thing being demonstrated here?

Reply:
Same for following examples.